microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/utils/AnnotationMemberInfo.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4mp.jdt.core.utils;
+
+import org.eclipse.lsp4j.Range;
+
+/**
+ * Utility class to store the annotation member info such as the value and the
```

Comment:
Add 2 samples inside <code> and explains the value and range result by using `|` to know the position:

 * one sample with one member value (ex : `@ConfigProperty(name = "greeti|ng.message") `
 * a second sample with 2 members value `@ConfigProperty(name = "greeting.message",defaultValue="he|llo") `

Reply:
The first example is valid, however I believe the second example doesn't apply, since this is only one member value and range.

Reply:
The second sample should return the range and value of defaultValue, no?

Reply:
Oh i see what you mean, yes you are right